randobot/handler.py

```diff
@@ -556,30 +604,58 @@ async def ex_settings(self, args, message):
+                    f"{' | '.join(minor_pool.keys())}"
                 )
                 await self.send_message(
                     'Use !settings <setting> to view available values.'
                 )
             # List available values for specific setting
-            elif len(args) == 1 and args[0] in draft.get('available_settings').get('minor').keys():
-                setting = draft.get('available_settings').get('minor').get(args[0])
+            elif len(args) == 1 and args[0] in minor_pool.keys():
+                setting = minor_pool.get(args[0])
                 await self.send_message(
                     f'Available values for {args[0].capitalize()}: {", ".join(value for value in setting.keys())}'
                 )
         elif draft.get('status') == 'complete':
             if len(args) == 0:
-                setting = draft.get('drafted_settings').get('bans')
                 await self.send_message(
-                    f"Bans for this race: {', '.join(value.capitalize() for value in setting.keys())}"
+                    f"Bans for this race: {', '.join(value.capitalize() for value in bans.keys())}",
+                    pinned=True
                 )
                 await self.send_message(
-                    f'Picks for this race:'
+                    'Picks for this race: ' + ', '.join(f"{key.capitalize()}: {value.capitalize()}" for key, value in picks.items()),
+                    pinned=True
                 )
-                for key, value in draft.get('drafted_settings').get('picks').items():
-                    await self.send_message(
-                        f'{key.capitalize()}: {value.capitalize()}'
-                    )
+                await self.send_message(
+                    '@entrants, settings for the race are pinned above.'
+                )
+        elif draft.get('status') == 'seed_rolled':
+            if draft.get('auto_draft'):
+                await self.send_message(
+                    'Picks for this race: ' + ', '.join(f"{key.capitalize()}: {value.capitalize()}" for key, value in picks.items()),
+                    pinned=True
+                )
+                await self.send_message(
+                    '@entrants, settings for the race are pinned above.'
+                )
+                return
+            # Delay settings reveal for 10 minutes after rolling the seed for qualifier races
+            if datetime.datetime.now() - draft.get('rolled_at') > datetime.timedelta(minutes=10):
+                await self.send_message(
+                    'Picks for this race: ' + ', '.join(f"{key.capitalize()}: {value.capitalize()}" for key, value in picks.items()),
+                    pinned=True
+                )
+                await self.send_message(
+                    '@entrants, settings for the race are pinned above.'
+                )
+                if not self.data.get('status').get('value') == 'invitational':
```

Comment:
I suggest using `if self.data.get('status').get('value') == 'open'` specifically so you don't try and set_invitational in the odd case that the race has already started somehow.

(also in general, use `x != y` rather than `not x == y`)